trueusd/README.md

```diff
@@ -1,11 +1,19 @@
 # Chainlink External Adapter for trueusd
 
+## Total Supply API
+
+https://core-api.real-time-attest.trustexplorer.io/trusttoken/TrueUSD
+
 ## Input Params
 
-_No params_
+- `field`: Required, the data field to return. Must be in camel-case style.
```

Comment:
Please document the different options here. I guess they are `(totalTrust|totalToken)`?

Reply:
Also maybe document what different options mean. I guess `totalTrust` are bank locked USD, and `totalToken` would be the total supply of blockchain issued `TUSD`?

Going forward we should invest more time in writing great documentation to make our adapters easy to understand and straightforward to use.